VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionsList.razor

```diff
@@ -41,9 +41,18 @@
     }
 </div>
 
-<Modal Title="Transaction Details" Visible="@txDetailsVisible" OnOk="@CloseTxDetails" OnCancel="@CloseTxDetails">
-    <VENFTApp_Blazor.Components.Account.TransactionDetails TxId="@txDetailsTxId" SubAccount="@SubAccount" Address="@Address" />
-</Modal>
+<VENFTApp_Blazor.Components.Modal.Modal @bind-Visible="@txDetailsVisible">
+    <ModalHeader>
+        <ModalTitle>Transaction Details</ModalTitle>
+    </ModalHeader>
+    <ModalBody>
+        <VENFTApp_Blazor.Components.Account.TransactionDetails TxId="@txDetailsTxId" SubAccount="@SubAccount" Address="@Address" />
+    </ModalBody>
+    <VENFTApp_Blazor.Components.Modal.ModalFooter>
+        <ModalButton Kind="ModalButtonKind.Cancel">Cancel</ModalButton>
+        <ModalButton OnClick="@context.Cancel">Ok</ModalButton>

```

Comment:
`@context.Cancel` will invoke the same function as Cancel button.
I think we should redesign some modals that don't need both `OK` and `Cancel` buttons.

Reply:
yes. agree. feel free to remove OK or Cancel from modals where users not need it. For example Save Private Key or VENFT Backup File Download modals are confusing a lot.